v63003/u_inref/gtm8787.exp

```diff
@@ -0,0 +1,34 @@
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+set timeout 60
+
+spawn /usr/local/bin/tcsh -f
+
+expect_after {
+	timeout { timeout_procedure }
+}
+
+proc timeout_procedure { } {
+	puts "timeout occurred"
+	exit -1
+}
+
+
+# Have columns higher than 80 as that can cause test failures on lines that are just above 80 columns in length
+send -- "stty cols 132\r"
+expect "stty cols 132\r"
+
+expect -exact ">"
+send -- "\$MUPIP Journal -Extract=-stdout -forward mumps.mjl & \r"
+expect -exact "x(10)"
```

Comment:
We are doing just 10 updates and expecting to see x(10) before exiting the expect script (i.e. Xing the terminal). I would prefer we do 1000 updates instead of 10. But please confirm that with this, we are still able to reproduce the KILLBYSIG error in V63002.